srtcore/api.cpp

```diff
@@ -1908,6 +2061,18 @@ SRTSOCKET CUDT::socket()
    }
 }
 
+int CUDT::setError(const CUDTException& e)
```

Comment:
One of them has been moved here from the header file - due to the reason of solving the header dependency problem. The other one is useful for quick returning a failure.